engine/atf_result.cpp

```diff
@@ -625,13 +625,21 @@ engine::calculate_atf_result(const optional< process::status >& body_status,
     try {
         result = atf_result::load(results_file);
     } catch (const engine::format_error& error) {
-        result = atf_result(atf_result::broken, error.what());
+        if (body_status) {
+            result = atf_result(atf_result::broken,
+                F("Error: %s. Test case %s") %
+                error.what() %
+                format_status(body_status.get()));
+        } else {
+            // The test case timed out.  apply() handles this case later.
```

Comment:
This seems a bit strange style-wise, but is already like this elsewhere so being consistent makes sense for now.

Reply:
Yes. The atf/kyua/lutok coding style doesn't match FreeBSD style(9).